newgrf/24251d2c/global.yaml

```diff
@@ -3,6 +3,3 @@ replaced-by:
   unique-id: "24252c2c"
 description: "A lot of trucks used in Hungary, from 1950 to 2010!"
```

Comment:
Ah seems some newgrf mention region in description without using tags.

Reply:
Plenty do, and with any new system, people will have to adjust to it. So it will be a while before most GRFs have a region assigned. But that is just part of the game.